bsp/qemu-sifive-e31-customer/settings.mk

```diff
@@ -0,0 +1,13 @@
+# Copyright (C) 2020 SiFive Inc
+# SPDX-License-Identifier: Apache-2.0
+
+RISCV_ARCH = rv32imac
+RISCV_ABI = ilp32
+RISCV_CMODEL = medlow
+RISCV_SERIES = None
+
+TARGET_TAGS = qemu
+TARGET_DHRY_ITERS = 20000000
+TARGET_CORE_ITERS = 5000
+TARGET_FREERTOS_WAIT_MS = 1000
+TARGET_INTR_WAIT_CYCLE  = 0
```

Comment:
add newline ;)

Reply:
_settings.mk_ is generated by _update-targets.sh_ and it doesn't have new line at the end of file.
I've made the fix, but the CI tests will fail. I will revert my change back.